devfile.yaml

```diff
@@ -1,26 +1,44 @@
 specVersion: 0.0.1
 name: chectl
 projects:
-  - name: chectl
+  - 
+    name: chectl
     source:
       type: git
       location: 'https://github.com/che-incubator/chectl.git'
 components:
-  - alias: theia-ide
+  - 
+    id: eclipse/che-theia/next
     type: cheEditor
-    id: eclipse/che-theia/1.0.0
-  - type: chePlugin
+  - 
     id: eclipse/che-machine-exec-plugin/0.0.1
+    type: chePlugin
+  - 
+    alias: git-devtools
+    type: dockerimage
+    image: sunix/git-devtools
+    mountSources: true
+    memoryLimit: 256M
+  - 
+    id: redhat/vscode-yaml/0.4.0
+    type: chePlugin
+  - 
+    alias: ts
+    id: che-incubator/typescript/1.30.2
+    type: chePlugin
+    memoryLimit: 1Gi 
+
 commands:
   - name: build
     actions:
       - type: exec
-        component: theia-ide
+        component: ts
```

Comment:
It would be ok to use `ts` component if you hadn't included a `devtools` component as well. Since you have added a `devtool` component, and since a user will mostly use that component to open a terminal, shouldn't we target the `devtool` component to run commands?

Reply:
it was not a devtools but a git-devtools == git related development tools

Reply:
@sunix it doesn't make sense to have separate containers for every development tool. Mainly because users will need to open a new terminal for every tool they use.